lib/fetchContent.js

```diff
@@ -348,6 +348,227 @@ function getArticleRelations(days, facets, aspects)
 		});
 }
 
+function getArticlesAggregation(days, facets, aspects, minCorrelation=2)
+{
+	const date = time.getDatetimeRange('days', days, 0);
+	aspects = checkMetaDataPresent(aspects);
+	const params = {
+		queryString: `lastPublishDateTime:> ${date.next}`,
+		maxResults : 100,
+		aspects : aspects,
+		facets : {
+			"names": facets,
+			"maxElements":-1
+		}
+	};
+
+	return searchDeeper(params, 10)
+		.then(results =>  extractArticles(results) )
+		.then(articles => aggregateArticles(articles, facets, minCorrelation, days, params, filterByMetadataCsv=null) )
```

Comment:
no need to add `filterByMetadataCsv` as the function handles null as a default value

Reply:
self-documenting code, innit.
I don't like leaving anonymous nulls in param lists.

Reply:
Defies the purpose of having a default value in the function declaration, then. Also I would create a var outside  and then pass in `filterByMetadataCsv`. Variable assignment within function call isn't great.

Reply:
tidied